src/components/confirm-dialog/ConfirmDialog.tsx

```diff
@@ -41,15 +45,15 @@ const ConfirmDialog: FC<ConfirmDialogProps> = ({
       <IconButton onClick={onDismiss} sx={styles.icon}>
         <CloseIcon />
       </IconButton>
-      <DialogContent dividers sx={styles.content}>
+      <DialogContent sx={styles.content}>
         <Typography variant='subtitle1'>{t(message)}</Typography>
       </DialogContent>
       <DialogActions sx={styles.actions}>
         <Button onClick={onConfirm} size='large' variant='tonal'>
```

Comment:
Move styles to styles file.

Reply:
yeah, use variant as typography in styles, and use enums for variants and size everywhere

Reply:
fixed